CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 #### Improvements
 * Fix #2717: Remove edit() methods from RawCustomResourceOperationsImpl taking InputStream arguments
 
+* Fix #2757: add `storage` and `served` to `Version` annotation
```

Comment:
We also need to add an entry about the CRD generator.
It might be best to mark it as an early preview so that users are warned.

Reply:
Done